src/algorithms/standard/tensortovectorreal.cpp

```diff
@@ -66,6 +68,11 @@ AlgorithmStatus TensorToVectorReal::process() {
     _timeStamps = tensor.dimension(2);
     _featsSize = tensor.dimension(3);
 
+    if (_channels != 1 && !_warned) {
+        E_WARNING("TensorToVectorReal: The channel axis (dimension 1) of the input tensor has size larger than 1, but the output of this algorithm is 2D. The batch, channel, and time axes (dimensions 0, 1, 2) will be flattened to the first dimension of the output matrix.");
```

Comment:
We output a vector of vector of reals, so the "matrix" terminology may be misleading.